perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -989,6 +989,42 @@ public void removeMember(PerunSession sess, Group group, Member member) throws I
 		}
 	}
 
+	@Override
+	public void removeMembers(PerunSession sess, Group group, List<Member> members) throws InternalErrorException, NotGroupMemberException, GroupNotExistsException {
+		// Check if the group is NOT members or administrators group
+		if (group.getName().equals(VosManager.MEMBERS_GROUP)) {
```

Comment:
It is not important here, but for the future work it is safer to use equals on non-null subject first like:
`VosManager.MEMBERS_GROUP.equals(group.getName())`
or use Object.equals() method to prevent NullPointerException. But in this specific case, group should be always filled with not-null name.